Explorer/Assets/DCL/UI/MainUIContainer/MainUIController.cs

```diff
@@ -0,0 +1,136 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL.Chat;
+using DCL.Minimap;
+using DCL.SidebarBus;
+using DCL.UI.Sidebar;
+using DG.Tweening;
+using JetBrains.Annotations;
+using MVC;
+using System;
+using System.Threading;
+using UnityEngine;
+using Utility;
+
+namespace DCL.UI.MainUI
+{
+    public class MainUIController : ControllerBase<MainUIView>
```

Comment:
We are lacking of test coverage on the newly created controllers, either this one, NotificationsMenuController, SidebarController, etc.. Consider on making proper test coverage on most important flows, or operations that might be valuable to cover.

Reply:
adding an issue as tech debt, we need to merge this PR and another to start testing the full flow of quests